repose-aggregator/src/docs/asciidoc/filters/index.adoc

```diff
@@ -2,15 +2,104 @@
 
 include::../_includes/in-progress.adoc[]
 
-<<add-header.adoc#,Add Header Filter>>::
-    Enables users to add new headers and header values to a request and/or response.
+== <<add-header.adoc#,Add Header Filter>>
+Enables users to add new headers and header values to a request and/or response.
 
-<<cors.adoc#,CORS Filter>>::
-    Enables users to add support for Cross-Origin Resource Sharing requests.
+== <<api-validator.adoc#,API Validator Filter>>
+*IN PROGRESS*
 
-<<rackspace-auth-user.adoc#,Rackspace Auth User Filter>>::
-    For identity usage for rate limiting and user access events.
+== <<body-extractor-to-header.adoc#,Body Extractor to Header Filter>>
+*IN PROGRESS*
 
-<<scripting.adoc#,Scripting Filter>>::
-    Enables users to write custom filters for Repose using a
-    variety of scripting languages.
+== <<body-patcher.adoc#,Body Patcher Filter>>
+*IN PROGRESS*
+
+== <<compression.adoc#,Compression Filter>>
+*IN PROGRESS*
+
+== <<content-type-stripper.adoc#,Content Type Stripper Filter>>
+*IN PROGRESS*
+
+== <<cors.adoc#,CORS Filter>>
+Enables users to add support for Cross-Origin Resource Sharing requests.
+
+== <<derp.adoc#,Delegation Response Processor (DeRP) Filter>>
+*IN PROGRESS*
+
+== <<destination-router.adoc#,Destination Router Filter>>
+*IN PROGRESS*
+
+== <<echo.adoc#,Echo Filter>>
+*IN PROGRESS*
+
+== <<exception.adoc#,Exception Filter>>
+*IN PROGRESS*
+
+== <<forwarded-proto.adoc#,Forwarded Protocol Filter>>
+*IN PROGRESS*
+
+== <<header-normalization.adoc#,Header Normalization filter>>
+*IN PROGRESS*
+
+== <<header-translation.adoc#,Header Translation Filter>>
+*IN PROGRESS*
+
+== <<header-user.adoc#,Header User Filter>>
+*IN PROGRESS*
+
+== <<herp.adoc#,Highly Efficient Record Processor (HERP) Filter>>
+*IN PROGRESS*
+
+== <<ip-user.adoc#,IP User Filter>>
+*IN PROGRESS*
+
+== <<iri-validator.adoc#,IRI Validator Filter>>
+*IN PROGRESS*
+
+== <<keystone-v2.adoc#,Keystone V2 Filter>>
+*IN PROGRESS*
+
+== <<keystone-v2-basic-auth.adoc#,Keystone v2 Basic Auth Filter>>
```

Comment:
Yeah...  It seems like no matter what, something will be "off".

In the Gradle file, there really isn't any filter context in my mind, so it should be completely alphabetical.  On the Filter doc page, the context is filters, so really it's fluff that should be ignored (both visually and in terms of sorting).  But, then those two lists would be sorted inconsistently with each other.  Also, that would mean that we shouldn't include the word "Filter" in each link on the Filters page as it's totally redundant, but that also feels wrong as "filter" is a part of the name of each filter.

I suppose we went down the path of redundancy a long time ago.
e.g.
`org.openrepose.filters.cors.CorsFilter`